release/edgex-go.yaml

```diff
@@ -1,9 +1,10 @@
 ---
 name: 'edgex-go'
-version: '1.3.0'
+version: '1.3.1'
 releaseName: 'hanoi'
-releaseStream: 'master'
+releaseStream: 'hanoi'
 repo: 'https://github.com/edgexfoundry/edgex-go.git'
+commitId: 'b46b7e8111e4043173f976c96fadf622ec12f3cd'
```

Comment:
Note that @tonyespy has a pending snaps PR on the `hanoi` branch waiting on a response to my question.
https://github.com/edgexfoundry/edgex-go/pull/3102

Reply:
@bill-mahoney , with the merge of @tonyespy snap PR the commit for `hanoi` release is now `12a188d7528ee379d18cf3e1ac715e90dfef7258`: